### PR TITLE
Add docs for relay API endpoint

### PR DESCRIPTION
**Pull request by ylv-io**

### CODE REVIEW

File: docs/modules/ROOT/pages/relay.adoc [hunk 1]
```diff
@@ -98,6 +98,16 @@ In addition to sending transactions, the Relayer can also sign arbitrary message
 const signResponse = await relayer.sign({ message });
 ----
 
+[[relayer-info]]
+=== Relayer Info
+
+A relayer's address can be retrieved using the `getAddress` method. Checkout the `getRelayer` method of the client to get more info about a relayer.
```
Comment (spalladino) [inline docs/modules/ROOT/pages/relay.adoc:104]: I can't find the getAddress method on the relayer object. Maybe you mixed up with the ethers signer object instead?

https://github.com/OpenZeppelin/defender-client/blob/master/src/relayer.ts

**Review by spalladino — COMMENTED** [review]
I think there may be a mixup with the method. Can you check?

File: docs/modules/ROOT/pages/relay.adoc (hunk 1 above)
Comment (ylv-io) [inline docs/modules/ROOT/pages/relay.adoc]: It is a poor choice of words.

Comment (ylv-io) [reply docs/modules/ROOT/pages/relay.adoc]: It is a poor choice of words.

File: docs/modules/ROOT/pages/relay.adoc [hunk 2]
```diff
@@ -98,6 +98,34 @@ In addition to sending transactions, the Relayer can also sign arbitrary message
...
 ----
 
+[[relayer-info]]
+=== Relayer Info
+
+A relayer's address can be retrieved using the `getAddress` method of the `DefenderRelaySigner` class.
+
+[source,jsx]
+----
+const addr = await relayer.getAddress();
```
Comment (spalladino) [inline docs/modules/ROOT/pages/relay.adoc:108]: Ah, I understand now what you meant. Still, in the examples right before the one you added, `relayer` refers to a `Relayer` object, not to a `DefenderRelaySigner`. I'd rename this one to `signer` for clarity.

```suggestion
const addr = await signer.getAddress();
```

Comment (spalladino) [reply docs/modules/ROOT/pages/relay.adoc:108]: @ylv-io bump!

**Pull request merged**